.spelling

```diff
@@ -1,182 +1,178 @@
-# markdown-spellcheck spelling configuration file
```

Comment:
Because I don't think it's necessary tbh, since we don't use any comments. Seems clearer to just have the file contain ordered entries and nothing else at this point :)